File: chatwoot/ruby/src/UpdateAnIntegrationsHookExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
    # config.api_key["api_access_token"] = "AGENT_BOT_API_KEY"
    # config.api_key["api_access_token"] = "PLATFORM_APP_API_KEY"
end

integrations_hook_update_payload = ChatwootClient::IntegrationsHookUpdatePayload.new

begin
    response = ChatwootClient::IntegrationsApi.new.update_an_integrations_hook(
        0, # account_id
        0, # hook_id
        integrations_hook_update_payload, # data
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling IntegrationsApi#update_an_integrations_hook: #{e}"
end

File: chatwoot/python/src/UpdateAnIntegrationsHookExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"userApiKey": "USER_API_KEY"},
    # api_key={"agentBotApiKey": "AGENT_BOT_API_KEY"},
    # api_key={"platformAppApiKey": "PLATFORM_APP_API_KEY"},
)

with ApiClient(configuration) as api_client:
    integrations_hook_update_payload = models.IntegrationsHookUpdatePayload(
    )

    try:
        response = api.IntegrationsApi(api_client).update_an_integrations_hook(
            account_id=0,
            hook_id=0,
            data=integrations_hook_update_payload,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling IntegrationsApi#update_an_integrations_hook: %s\n" % e)
